### DISCUSSION TITLE
Is there a way to add validation based on the equality of two fields.

### DISCUSSION THREAD

**Discussion by venkateshpullaganti** [manojadams/mui-forms#81] [Q&A] [answered]
Is there a way to add validation for phone and alternate phone should not be same?

**Comment by manojadams** [comment, marked as answer]
@venkateshpullaganti It can be done using custom fields.

**Example:**
Use the following example and try to book a flight with same departure date and return date:
https://mui-forms.vercel.app/examples/booking-form
